samples/servicecontrol/adapter-rabbitmq-different-topologies/sample.md

```diff
@@ -33,12 +33,12 @@ NOTE: In order to connect to a different RabbitMQ broker, ensure all connection
 
 ## Running the project
 
- 1. Start the projects: Adapter, Sales and Shipping (right-click on the project, select the `Debug > Start new instance` option). Make sure adapter starts first because on start-up it creates a queue that is used for heartbeats.
+ 1. Start the projects: Adapter, Sales and Shipping (right-click on the project, select the `Debug > Start new instance` option). Ensure the adapter starts first because on start-up it creates a queue that is used for heartbeats.
```

Comment:
What do we do about this sentence? As @SimonCropp pointed out samples should be <kbd>F5</kbd> ready. And in my tests the above is not required: if the startup order is not correct endpoints will emit warnings because heartbeats cannot be delivered, all warnings will go away as soon as the adapter starts and Qs are created.